### ISSUE TITLE
LuraToon: Add support for deep links

### ISSUE THREAD

**Issue by gamespvp93** [keiyoushi/tachiyomi-extensions-source#3020]
### Source name

LuraToon

### Source language

Brazilian portuguese

### Describe your suggested feature

When the person clicks on the website link, they won't be redirected to the extension, and this is because the extension doesn't have these print functions blocked, unlike other extensions. Additionally, the other extensions not only are unblocked but also allow the person to change the extension's functionality. Whenever someone clicks on the website link, they will be directed to the application.

### Other details

![Screenshot_20240514-132039](https://github.com/keiyoushi/extensions-source/assets/28785851/4865d4f7-db90-4bd9-8c6b-1d6e8234d8b0)
![Screenshot_20240514-132108](https://github.com/keiyoushi/extensions-source/assets/28785851/b5ebf61e-3304-4819-90b9-c39f350fa59d)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Norsze** [comment]
/edit-title "LuraToon: Add support for deep links"
